docs/resources/ecosystem.md

```diff
@@ -0,0 +1,21 @@
+---
+id: ecosystem
+title:  "Ecosystem"
+---
+
+## Official ZIO Libraries
+
+- [ZIO Actors](https://github.com/zio/zio-actors): A high-performance, purely-functional library for building, composing, and supervising typed actors based on ZIO
+- [ZIO Kafka](https://github.com/zio/zio-kafka): A Kafka client for ZIO and ZIO Streams
+- [ZIO Keeper](https://github.com/zio/zio-keeper): A functional library for consistent replication of metadata across dynamic clusters
+- [ZIO SQS](https://github.com/zio/zio-sqs): A ZIO-powered client for AWS SQS
+
+## Other Libraries with direct ZIO Support
+
+- [cakeless](https://github.com/itkpi/cakeless): Better reader monad for deeply-nested cakes
+- [izumi](https://github.com/7mind/izumi): Productivity-oriented collection of lightweight fancy stuff for Scala toolchain
+- [scanamo](https://github.com/scanamo/scanamo): Simpler DynamoDB access for Scala
+- [sttp](https://github.com/softwaremill/sttp): The Scala HTTP client you always wanted!
+- [zio-slf4j](https://github.com/NeQuissimus/zio-slf4j): Referentially transparent logging with slf4j
+
+Don't hesitate to create a PR to add your own library!
```

Comment:
I'd add a whole section on `ZIO Compatible Libraries`, and list:

 - Cats
 - Scalaz
 - Doobie
 - Http4s
 - FS2
 - Circe
 - Etc.

All the mainstream functional Scala libraries. In fact, maybe this section should be the top one in this file, because it's the leading question.

Reply:
Makes sense. I wasn't sure if I should list the whole cats-effect ecosystem, but yeah at least the most important ones.